sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -95,6 +158,17 @@ export interface DetectLanguageSuccessResult extends TextAnalyticsSuccessResult
 // @public
 export type DocumentSentimentLabel = "positive" | "neutral" | "negative" | "mixed";
 
+// @public (undocumented)
+export type EntitiesTask = {
+    parameters?: EntitiesTaskParameters;
```

Comment:
Can we flatten this too? Why not

```ts
export interface EntitiesTask {
   modelVersion?: string
}
```

What does having a single subkey called `parameters` get us? Is this expected to grow considerably?

Reply:
Fixed. For context: this was supposed to be a thin layer around generated code and the generated code has this `parameters` thingy and I did not think too much about it.